models/index.js

```diff
@@ -0,0 +1,34 @@
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || "development";
+const config = require(`${__dirname}/../config/config.json`)[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
```

Comment:
load JS dynamically which make depedency  impossibele to trace, should be prevented

Reply:
Resolved.

Reply:
mmm looks like is not changed here

Reply:
Resolved.